app/cypress/e2e/0-ndr-core-tests/gp_user_workflows/view_lloyd_george_is_bsol_workflow.cy.js

```diff
@@ -216,15 +216,9 @@ describe('GP Workflow: View Lloyd George record', () => {
             () => {
                 beforeEachConfiguration(Roles.GP_ADMIN);
 
-                let request = 0;
-                const replies = [
-                    { statusCode: 200, body: viewLloydGeorgePayload },
-                    { statusCode: 404 },
-                ];
-
-                cy.intercept('GET', '/LloydGeorgeStitch*', (req) => {
-                    req.reply(replies[request]);
-                    request = request + 1;
+                cy.intercept('GET', '/LloydGeorgeStitch*', {
+                    statusCode: 200,
+                    body: viewLloydGeorgePayload,
```

Comment:
Can we add a test where the first call to the stitch return 200 and then delete the record and check there is another call to the stitch lambda?